stack.yaml

```diff
@@ -14,241 +14,3 @@ ghc-options:
 packages:
 - .
 extra-deps:
-- Cabal-syntax-3.10.3.0
-- Glob-0.10.2
-- HUnit-1.6.2.0
-- JuicyPixels-3.3.9
-- OneTuple-0.4.2
-- Only-0.1
-- QuickCheck-2.15.0.1
-- SHA-1.6.4.4
-- StateVar-1.2.2
-- aeson-2.2.3.0
-- aeson-pretty-0.8.10
-- alex-3.5.1.0
-- ansi-terminal-1.1.1
-- ansi-terminal-types-1.1
-- appar-0.1.8
-- array-0.5.6.0
-- asn1-encoding-0.9.6
-- asn1-parse-0.9.5
-- asn1-types-0.3.4
-- assoc-1.1.1
-- async-2.2.5
-- attoparsec-0.14.4
```

Comment:
not quite sure why these got removed.

Reply:
Something got borked :shrug: Anyway, thanks, I'll have a look.

Reply:
So... apparently either you don't have `sed`, it doesn't behave like the makefile expects it to, or `cabal.project.freeze` was somehow missing. No idea tbh. "it works fine on my machine" :sweat_smile: